api/v3/CustomValue/Gettreevalues.php

```diff
@@ -143,7 +143,15 @@ function civicrm_api3_custom_value_gettreevalues(array $params) {
     $treeParams['entityType'] = 'AwardsCaseTypes';
   }
 
-  $groupID = !empty($params['custom_group.id']) ? $params['custom_group.id'] : NULL;
+  if (!empty($params['custom_group.name']) && !is_array($params['custom_group.name'])) {
+    $result = civicrm_api3('CustomGroup', 'getsingle', [
+      'return' => ['id'],
+      'name' => $params['custom_group.name'],
+    ]);
```

Comment:
getsingle will throw an exception if the group is not found. So we need to put this in a try catch block

Reply:
Also can you add as part of technical details the changes to the Customvalues.gettreevalues API?